arm/Microsoft.Synapse/privateLinkHubs/readme.md

```diff
@@ -0,0 +1,115 @@
+# Synapse PrivateLinkHubs `[Microsoft.Synapse/privateLinkHubs]`
+
+This module deploys a Synapse Private Link Hubs.
+
+## Resource Types
+
+| Resource Type                                             | API Version        |
+| :-------------------------------------------------------- | :----------------- |
+| `Microsoft.Authorization/locks`                           | 2016-09-01         |
+| `Microsoft.Authorization/roleAssignments`                 | 2020-04-01-preview |
+| `Microsoft.Network/privateEndpoints`                      | 2021-03-01         |
+| `Microsoft.Network/privateEndpoints/privateDnsZoneGroups` | 2021-03-01         |
+| `Microsoft.Synapse/privateLinkHubs`                       | 2021-06-01         |
+
+## Parameters
+
+| Parameter Name     | Type   | Default Value                | Possible Values                          | Description                                                                                                                                                                                                                                                                                                                                                                                                    |
+| :----------------- | :----- | :--------------------------- | :--------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I guess you have the markdown extension set to autoformat tables. Please undo the excess spaces and '-'. These files become unreadable if you don't open them locally

Reply:
There is a settings for that in the extension

Reply:
Can we add that setting to .vscode settings.json file?